windows/micropython.vcxproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Project templates/settings change, not always in a compatible way. It's not a bug, it's a breaking change. In this case, all settings being exposed in the vcxproj of this project appear to be compatible between versions and the transition is seamless. Not all project types are the same, and the breaking changes are specially common in mobile application project types or unstable/beta types such as pyproj.